posix.mak

```diff
@@ -279,7 +284,14 @@ install :
 	$(MAKE) -f $(MAKEFILE) OS=$(OS) MODEL=$(MODEL) BUILD=release INSTALL_DIR=$(INSTALL_DIR) \
 		DMD=$(DMD) install2
 
+.PHONY : unittest
+ifeq (1,$(BUILD_WAS_SPECIFIED))
 unittest : $(addsuffix .d,$(addprefix unittest/,$(D_MODULES)))
+else
+unittest : unittest-debug unittest-release
+unittest-%:
+	$(MAKE) $(MAKEFLAGS) unittest -f $(MAKEFILE) OS=$(OS) MODEL=$(MODEL) DMD=$(DMD) BUILD=$*
```

Comment:
Ok, I got it working locally:
- ```.PHONY : unittest``` is unnecessary.
- I changed this line to: ```$(MAKE) -f $(MAKEFILE) $(MAKEFLAGS) unittest BUILD=$*```

The order of the ```make``` arguments is important: ```$(MAKEFLAGS)``` needs to go after ```-f $(MAKEFILE)```.
(I omitted ```DMD```, ```OS```, and ```MODEL``` because they were unnecessary.)